src/Uno.UI.Tests/BinderTests/Given_Binder.cs

```diff
@@ -865,6 +865,22 @@ public void When_SelfRelativeSource()
 			Assert.AreEqual(42, SUT.Property2);
 		}
 
+#if !NET461
```

Comment:
The unit tests are `NET461` only... Why do you have this condition?

Reply:
The explicit method binding that gets triggered is !Net461 only

Reply:
guess I can't unittest it.

Reply:
What exactly is missing for this? It can be added to `NET461`. The databinding layer should be fully coverable by unit tests.